JournalMonitor/JournalMonitor.cs

```diff
@@ -139,7 +138,7 @@ public static List<Event> ParseJournalEntry(string line, bool fromLogLoad = fals
                                     bool wanted = JsonParsing.getOptionalBool(data, "Wanted") ?? false;
                                     bool activeFine = JsonParsing.getOptionalBool(data, "ActiveFine") ?? false;
 
-                                    events.Add(new DockedEvent(timestamp, systemName, systemAddress, marketId, stationName, stationState, stationModel, allegiance, faction, factionState, Economies, government, distancefromstar, stationServices, cockpitBreach, wanted, activeFine) { raw = line, fromLoad = fromLogLoad });
+                                    events.Add(new DockedEvent(timestamp, systemName, systemAddress, marketId, stationName, stationState, stationModel, allegiance, faction.name, faction.FactionState, Economies, government, distancefromstar, stationServices, cockpitBreach, wanted, activeFine) { raw = line, fromLoad = fromLogLoad });
```

Comment:
Consider passing faction as an object? We're reconstituting the object at EDDI.cs line 1027. We can simplify if we build the object here and simply pass the object.